README.md

```diff
@@ -108,3 +104,27 @@ jobs:
 If you need to use an in-development version of these tools from a branch, you can update the `ref` in `repositories` section to point to a different branch.
 
 The `build-pipeline.yml` template has a few parameters. You can find out more about them in [build-pipeline.yml](https://github.com/shotgunsoftware/tk-ci-tools/blob/master/build-pipeline.yml).
+
```

Comment:
You can preview the documentation [here](https://github.com/shotgunsoftware/tk-ci-tools/blob/478ecd2afcfccc2af109a2a24c9f7debed19bbe7/README.md#create-a-build-in-azure-pipelines)